src/selectors/per-thread/markers.js

```diff
@@ -73,16 +63,30 @@ export function getMarkerSelectorsPerThread(threadSelectors: *) {
   const _getLastSampleTime: Selector<Milliseconds> = state =>
     threadSelectors.getThread(state).samples.time.slice(-1)[0] || 0;
 
-  const getMarkers: Selector<Marker[]> = createSelector(
+  const getDerivedMarkers: Selector<Marker[]> = createSelector(
```

Comment:
This should be `_getDerivedMarkers` as it's not exported. I also think a code comment would help out here to explain why it exists.